.travis.yml

```diff
@@ -67,7 +67,7 @@ script:
   - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
   - if [ "$TESTS" == true ]; then make test-ci ; else echo 'skipping tests'; fi
   - if [ "$PROTO_GEN_TEST" == true ]; then make proto && git diff --name-status --exit-code ; else echo 'skipping protoc validation'; fi
-  - if [ "$ALL_IN_ONE" == true ]; then bash ./scripts/travis/build-all-in-one-image.sh ; else echo 'skipping all_in_one'; fi
+  - if [ "$ALL_IN_ONE" == true ]; then make create-baseimg-debugimg && bash ./scripts/travis/build-all-in-one-image.sh ; else echo 'skipping all_in_one'; fi
```

Comment:
Why is the extra step `make create-baseimg-debugimg` needed here but not in DOCKER step in L73?
Also, why not call it from `./scripts/travis/build-all-in-one-image.sh`?

Reply:
> Why is the extra step make create-baseimg-debugimg needed here but not in DOCKER step in L73?
My understanding is that script calls: make docker -> make docker-images-only -> make docker-images-jaeger-backend-debug which has dependencies on create-baseimg and create-debugimg.

> Also, why not call it from ./scripts/travis/build-all-in-one-image.sh?
We can do it if you think that is the better place.

Reply:
Should probably be done after #2325, as it touches this code as well.

Reply:
if need to support multi arch for other components.

base images need to published with multi arch too. 
or just upgrade Dockerfile like https://github.com/morlay/jaeger/blob/master/cmd/all-in-one/Dockerfile